src/app/OperationalDeviceProxy.cpp

```diff
@@ -57,10 +58,36 @@ void OperationalDeviceProxy::MoveToState(State aTargetState)
     }
 }
 
+bool OperationalDeviceProxy::CheckAndLoadExistingSession()
```

Comment:
Are the actions indeed `Check` and `Load` ?

Reading through the code, this does a `finds the secure session for the current node` - effectively it is a call to `FindSecureSessionForNode` for CASE. Could we name it as such? Like `FindExistingCASESession` or similar?